sky/dashboard/src/components/jobs.jsx

```diff
@@ -77,6 +77,21 @@ const ALL_WORKSPACES_VALUE = '__ALL_WORKSPACES__';
 // Define constant for "All Users"
 const ALL_USERS_VALUE = '__ALL_USERS__';
 
+// Helper function to filter jobs by name
+export function filterJobsByName(jobs, nameFilter) {
```

Comment:
Should we add this filter for clusters as well for consistency

Reply:
Good catch, done.